src/reducers/assignment.js

```diff
@@ -13,7 +13,7 @@ export function assignment(state = initialState, action) {
     case types.CREATE_ASSIGNMENT:
       return { ...state,
         assignments: {
```

Comment:
The shape of state is unnecessarily deep: `assignment`>`assignments`>`data` should just be `assignments`>`data` or similar. @shaunanoordin